src/app/modules/test-navigator/user-activity-set.ts

```diff
@@ -76,6 +98,27 @@ export class CompositeUserActivitySet implements UserActivitySet {
     return types.length === 1 && types[0] === type;
   }
 
+  /**
+   * Provides a view of the activity set reduced to those associated with this node or with (sub-)nodes
+   * that are not currently visible in the tree.
+   * For example, if the given node is 'root', and the activities include some that are associated with
+   * node 'root/sub-node', then those activities will only be included if that node is currently *not*
+   * visible. The rational is that those activities will be displayed on that sub-node rather than this
+   * one.
+   * @param node a tree node providing the viewpoint for filtering the activities.
+   */
+  ownAndChildActivitiesWithNoVisibleCloserAncestorNode(node: TestNavigatorTreeNode): UserActivitySet {
```

Comment:
allActivitiesWithoutBetterFittingNodeForDisplayThan
or
withoutBetterFittingNodeForDisplayThan  (since method operates on the relevant activities)